docs/genai/deployment/deploy-serving-graph-as-job.ipynb

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.            image=self.image,
</pre><p>This is a concrete example rather than a test in a test suite, so a specific image should be provided or none at all (so it will take the default mlrun image).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/8900/discussion/#comment-2521623186'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>